src/firefly/js/visualize/MultiViewCntlr.js

```diff
@@ -126,49 +138,43 @@ export function dispatchRemoveViewer(viewerId) {
 /**
  *
  * @param {string} viewerId
- * @param {string[]} plotIdAry  array of plotIds
+ * @param {string[]} itemIdAry  array of itemIds
+ * @param {string} containerType a string with container type, IMAGE and PLOT2D are predefined
  *
  */
-export function dispatchAddImages(viewerId, plotIdAry) {
-    flux.process({type: ADD_IMAGES , payload: {viewerId, plotIdAry} });
+export function dispatchAddViewerItems(viewerId, itemIdAry, containerType) {
+    flux.process({type: ADD_VIEWER_ITEMS , payload: {viewerId, itemIdAry, containerType} });
 }
 
-/**
- *
- * @param {Array.<{plotId:string, requestAry:WebPlotRequest[]}>} imageAry
- *
- */
-export function dispatchAddToAutoReceiver(imageAry) {
-    flux.process({type: ADD_TO_AUTO_RECEIVER , payload: {imageAry} });
-}
 
 /**
  * 
  * @param {string} viewerId
  * @param {string} layout single or grid
  * @param {string} layoutDetail more detail about the type of layout, hint to UI
  */
-export function dispatchChangeLayout(viewerId, layout, layoutDetail) {
-    flux.process({type: CHANGE_LAYOUT , payload: {viewerId, layout, layoutDetail} });
+export function dispatchChangeViewerLayout(viewerId, layout, layoutDetail) {
+    flux.process({type: CHANGE_VIEWER_LAYOUT , payload: {viewerId, layout, layoutDetail} });
 }
 
 
 /**
  *
  * @param {string} viewerId
- * @param {string[]} plotIdAry array of string of plotId
+ * @param {string[]} itemIdAry array of string of itemId
  */
-export function dispatchRemoveImages(viewerId, plotIdAry) {
-    flux.process({type: REMOVE_IMAGES , payload: {viewerId, plotIdAry} });
+export function dispatchRemoveViewerItems(viewerId, itemIdAry) {
+    flux.process({type: REMOVE_VIEWER_ITEMS , payload: {viewerId, itemIdAry} });
 }
 
 /**
  *
  * @param {string} viewerId
- * @param {string[]} plotIdAry  array of {plotId : string, requestAry : array of WebPlotRequest}
+ * @param {string[]} itemIdAry array of string of itemId
+ * @param {string} containerType a string with container type, IMAGE and PLOT2D are predefined
  */
-export function dispatchReplaceImages(viewerId, plotIdAry) {
-    flux.process({type: REPLACE_IMAGES , payload: {viewerId, plotIdAry} });
+export function dispatchReplaceViewerImages(viewerId, itemIdAry, containerType) {
```

Comment:
We might want to rename it to dispatchReplaceViewerItems - as you did for other dispatchers